orchestrationSpecs/packages/migration-workflow-templates/src/workflowTemplates/workflowCommandOrchestrator.ts

```diff
@@ -0,0 +1,259 @@
+        expression: expr.cast(expr.literal("true")).to<string>(),
+        description: "Allow insecure connections to source cluster"
+    }),
+    sourceVersion: defineParam({ 
+        expression: expr.cast(expr.literal("")).to<string>(),
+        description: "Source cluster version (e.g. ES_5.6, OS_2.19)"
+    }),
+    targetConfig: defineParam({ 
+        expression: expr.cast(expr.literal("")).to<string>(),
+        description: "Target cluster config as JSON object fragment"
+    }),
+    snapshotConfig: defineParam({ 
+        expression: expr.cast(expr.literal("")).to<string>(),
+        description: "Snapshot config as JSON object fragment"
+    })
+};
+
+const WORKFLOW_CONFIGURE_AND_SUBMIT_SCRIPT = `
+set -e -x
+
+echo "Building and submitting migration workflow..."
+
+# Create migration config JSON
+cat > /tmp/migration_config.json << 'EOF'
+{
+  "sourceClusters": {
+    "source": {
+      "endpoint": "{{inputs.parameters.sourceEndpoint}}",
+      "allowInsecure": {{inputs.parameters.sourceAllowInsecure}},
+      "version": "{{inputs.parameters.sourceVersion}}",
+      "snapshotRepo": {{inputs.parameters.snapshotConfig}}
+    }
+  },
+  "targetClusters": {
+    "target": {{inputs.parameters.targetConfig}}
+  },
+  "migrationConfigs": [
+    {
+      "fromSource": "source",
+      "toTarget": "target",
+      "snapshotExtractAndLoadConfigs": [
+        {
+          "createSnapshotConfig": {
+            "s3RoleArn": "arn:aws:iam::123456789012:role/test-migration-role"
+          },
+          "snapshotConfig": {
+            "snapshotNameConfig": {
+              "snapshotNamePrefix": "rfs"
+            }
+          },
+          "migrations": [
+            {
+              "metadataMigrationConfig": {
+                "indexAllowlist": ["*"]
+              },
+              "documentBackfillConfig": {
+                "indexAllowlist": ["*"]
+              }
+            }
+          ]
+        }
+      ]
+    }
+  ]
+}
+EOF
+
+echo "Migration config contents:"
+cat /tmp/migration_config.json
+
+. /etc/profile.d/venv.sh
+source /.venv/bin/activate
+
+# Clear and load configuration
+echo "Clearing existing workflow configuration..."
+workflow configure clear --confirm
+
+echo "Loading configuration from JSON..."
+cat /tmp/migration_config.json | workflow configure edit --stdin
+
+# Submit workflow and capture the workflow name
+echo "Submitting workflow..."
+WORKFLOW_OUTPUT=$(workflow submit 2>&1)
+echo "Workflow submit output: $WORKFLOW_OUTPUT"
+
+# Extract workflow name (format: "  Name: workflow-name")
+WORKFLOW_NAME=$(echo "$WORKFLOW_OUTPUT" | grep "Name:" | awk '{print $2}' | tr -d '\n')
+
+if [ -z "$WORKFLOW_NAME" ]; then
+    echo "ERROR: Could not extract workflow name from output"
+    echo "Full output was: $WORKFLOW_OUTPUT"
+    exit 1
+fi
+
+echo "Workflow submitted successfully: $WORKFLOW_NAME"
+mkdir -p /tmp/outputs
+echo "$WORKFLOW_NAME" > /tmp/outputs/workflowName
+sync
+sleep 2
```

Comment:
Why do you need these two lines?  I haven't seen issues getting file contents as output variables.  It looks like your experience might have been different.